packages/cli-e2e/jest.config.ts

```diff
@@ -7,4 +7,10 @@ export default {
   testEnvironment: 'node',
   globalSetup: './setup.ts',
   globalTeardown: './teardown.ts',
+  /**
+   * TODO: CDX-160: Include spec files from ui projects once we have a mock of the headless engine
+   * In th meantmime, do not execute tests outside the __tests__ folder.
+   * More info -> https://jestjs.io/docs/configuration#testmatch-arraystring
+   */
```

Comment:
I'm not sure that should be part of the End-to-end tests. (and that those tests are end-to-end tests)

Reply:
That's fine, but I still need to update the `testMatch` parameter or these non-E2E tests will be executed